doc/coq.rst

```diff
@@ -32,35 +33,74 @@ The basic form for defining Coq libraries is very similar to the OCaml form:
      (synopsis <text>)
      (modules <ordered_set_lang>)
      (libraries <ocaml_libraries>)
+     (theories <coq_libraries>)
      (flags <coq_flags>))
 
-The stanza will build all `.v` files on the given directory. The semantics of fields is:
-
-- ``<module_prefix>>`` will be used as the default Coq library prefix ``-R``,
-- the ``modules`` field does allow to constraint the set of modules
-  included in the library, similarly to its OCaml counterpart,
-- ``public_name`` will make Dune generate install rules for the `.vo`
-  files; files will be installed in
-  ``lib/coq/user-contrib/<module_prefix>``, as customary in the
-  make-based Coq package eco-system. For compatibility, we also installs the `.cmxs`
-  files appearing in `<ocaml-librarie>` under the `user-contrib` prefix.
-- ``<coq_flags>`` will be passed to ``coqc``,
+The stanza will build all ``.v`` files on the given directory. The semantics of fields is:
+
+- ``<module_prefix>`` is a dot-separated list of valid Coq module
+  names and determines the module scope under which the theory is
+  compiled [``-R`` option]. For example, if ``<module_prefix>`` is
+  ``foo.Bar``, the theory modules will be named as
+  ``foo.Bar.module1``, ``foo.Bar.module2``, etc... Note that modules
+  in the same theory don't see the ``foo.Bar`` prefix, in the same
+  way that OCaml ``wrapped`` libraries do. For compatibility reasons,
+  the 1.0 version of the Coq language installs a theory named
```

Comment:
Doesn't exist yet.

Reply:
Thansk for the review @Zimmi48 ; I've added a note about 1.0.